examples/mongo_chat.py

```
@@ -0,0 +1,63 @@
+        chat_state = load_chat_history(session_id)
+    else:
+        print("Using default in-memory persistence for chat history.")
+        chat_state = {"session_memory": {}, "turns": [], "components": {}}
+
+    # Initialize the pipeline.
+    # Note: Pass the session_id to allow personalization context loading.
+    pipeline = TCAPipeline(mode="therapist", session_id=session_id)
+    pipeline.load(chat_state)
+
+    # Optionally load the vector store.
+    vector_store = load_vector_store()
```

Ruff (0.8.2)

Local variable `vector_store` is assigned to but never used

Remove assignment to unused variable `vector_store`

(F841)